device/protocol_mqtt_v1.go

```diff
@@ -297,6 +297,53 @@ func (d *Device) SendAggregateMessage(interfaceName, interfacePath string, value
 	return d.SendAggregateMessageWithTimestamp(interfaceName, interfacePath, values, time.Time{})
 }
 
+// SetProperty sets a  property in Astarte. It adds to the publishing channel a property set
+// message that is equivalent to `SendIndividualMessage` in every regard, except this call will fail
+// if the interface isn't a property interface. It is advised to use this call, as APIs might change in
+// the future. Once the property is set, it is also added to the local cache and can be retrieved at any time
+// using `GetProperty` or `GetAllProperties`.
+// This call can be blocking if the channel is full (see `MaxInflightMessages`).
+func (d *Device) SetProperty(interfaceName, path string, value interface{}) error {
+	// Get the interface from the introspection
+	iface, ok := d.interfaces[interfaceName]
+	if ok {
+		if iface.Type != interfaces.PropertiesType {
```

Comment:
Are we still allowing sending properties with `SendIndividualMessage`? If so, I'd also add some kind of "deprecation notice" in the documentation for that function so users can start using `SetProperty` for properties.

Reply:
I wanted to do this too, but for the sake of clarity and to avoid putting just too much on the table with this PR I wanted to defer this to a future PR